pkg/apis/bus/v1alpha1/actions.go

```diff
@@ -17,6 +17,7 @@ limitations under the License.
 package v1alpha1
 
 // Action is the action that Job controller will take according to the event.
+// +kubebuilder:validation:Enum=AbortJob;RestartJob;RestartTask;TerminateJob;CompleteJob;ResumeJob;SyncJob;EnqueueJob;SyncQueue;OpenQueue;CloseQueue
```

Comment:
`kubebuilder:validation:Enum` is a validation mechanism. It can ensure that the values entered by users are one of the predefined enumeration values and match the main version of volcano.
Other illegal inputs will lead to unpredictability during the runtime of volcano.

Reply:
I understand what you mean, and what I want to say is that we should just validate the fileds that exposed to users, because user may set an illegal value: )

Reply:
Volcano itself should make sure setting a valid value, otherwise when we add a new state we should also add the validation, which is easy to forget.

Reply:
> I understand what you mean, and what I want to say is that we should just validate the fileds that exposed to users, because user may set an illegal value: )

I agree. just validate the fileds that exposed to users.